IPython/core/completer.py

```diff
@@ -1410,7 +1410,7 @@ def _default_arguments(self, obj):
             pass
         elif not (inspect.isfunction(obj) or inspect.ismethod(obj)):
             if inspect.isclass(obj):
-                #for cython embededsignature=True the constructor docstring
+                #for cython embeddedsignature=True the constructor docstring
```

Comment:
This should actually say `embedsignature` - see the Cython docs here: http://cython.readthedocs.io/en/latest/src/reference/compilation.html